modules/ROOT/pages/database-administration/standard-databases/listing-databases.adoc

```diff
@@ -117,34 +119,42 @@ See <<database-states>> for more information.
 
 | default
 |
-Show if this is the default database for the DBMS. label:default-output[]
+`true` if this is the default database for the DBMS. label:default-output[]
 Not returned by `SHOW HOME DATABASE` or `SHOW DEFAULT DATABASE`.
 | BOOLEAN
 
 | home
 |
-Shown if this is the home database for the current user. label:default-output[]
+`true` if this is the home database for the current user. label:default-output[]
 Not returned by `SHOW HOME DATABASE` or `SHOW DEFAULT DATABASE`.
 | BOOLEAN
 
 | `currentPrimariesCount`
 | Number of primaries for this database reported as running currently.
 It is the same as the number of rows where `role=primary` and `name=this database`.
+
+The value for composite databases is `NULL` because it does not apply to them.footnote:compositeDb[This change applies to versions 2025.04 and later and 5.26.5 and later.]
```

Comment:
```suggestion
The value for composite databases is `NULL` because it does not apply to them.footnote:compositeDb[This change applies to versions 5.26.5 and later.]
```